x-pack/plugins/elastic_assistant/server/routes/user_conversations/find_route.ts

```diff
@@ -46,13 +46,16 @@ export const findUserConversationsRoute = (router: ElasticAssistantPluginRouter)
         try {
           const { query } = request;
           const ctx = await context.resolve(['core', 'elasticAssistant', 'licensing']);
-          const license = ctx.licensing.license;
-          if (!hasAIAssistantLicense(license)) {
-            return response.forbidden({
-              body: {
-                message: UPGRADE_LICENSE_MESSAGE,
-              },
-            });
+          // Perform license and authenticated user checks
```

Comment:
There are a lot more spots besides this where the pattern does not use `performChecks`. I'm going to fix in a follow up in order to avoid mass changes, unless someone thinks we should do it now